words/objects.txt

```diff
@@ -292,6 +388,7 @@ bubble
 bubbler
 buckaroo
 bucket
+buckthornpepperberry
```

Comment:
lol

Reply:
whoops. `/n` added

Reply:
Didn’t occur to me it was a missing newline! Just figured it was a very verbose berry

Reply:
well now I need to add verbose to the predicates list so that `verbose-berry` can be a name possibility!